src/source-updater.js

```diff
@@ -225,15 +226,19 @@ export default class SourceUpdater extends videojs.EventTarget {
     }
 
     if (codecs.audio) {
-      this.audioBuffer = this.mediaSource.addSourceBuffer(`audio/mp4;codecs="${codecs.audio}"`);
+      const mime = getMimeForCodec(codecs.audio);
```

Comment:
This function should automatically get the video mime type merged with codes. This should allow us to support whatever the browser does.